packages/next/src/client/components/react-dev-overlay/internal/components/copy-button/index.tsx

```diff
@@ -0,0 +1,42 @@
+export function CopyButton({
+  label,
+  content,
+  ...props
+}: {
+  label?: string
+  content: string
+  [key: string]: any
+}) {
+  return (
+    <span
+      {...props}
+      title={label}
+      aria-label={label}
+      role="button"
+      onClick={() => {
+        navigator.clipboard.writeText(content)
```

Comment:
should we provide feedback if this throws?

Reply:
The support seems pretty well, I feel it's fine since this is a minor feature on dev mode error overlay. We have CSS effects and users can also check the console.
![image](https://github.com/vercel/next.js/assets/4800338/17558bb8-f94c-4198-a417-add299c7cef2)

Reply:
Users could've disabled the Clipboard API though or not give permissions. Some feedback would be nice. Not just for error but finish feedback as well.

Reply:
Add a error logging when it's not available

Reply:
I was thinking more about feedback in the UI e.g. a small icon if it fails and another one if it passes.